__main__.py

```diff
@@ -60,13 +60,35 @@ def validate_project(platform, project, report_type):
         sys.exit(1)
 
 
+def args_to_list(platform, projects):
```

Comment:
if we wanted to get particular we could also add an else case where someone puts in a value other than those 3.  However, this to me seems adequate for this PR.